OrbitCore/LinuxUtils.cpp

```diff
@@ -149,6 +146,9 @@ void ListModules(pid_t pid,
     if (elf_file->GetBuildId().empty()) continue;
```

Comment:
do we really want to filter out these?

Reply:
Could you provide more details why you wouldn't want to filter these out?

Reply:
If a file is missing build id it does not necessarily mean we are unable to load symbols. If it has .symtab section available we do not really care if it has a build id. Also we might ending up providing user incomplete information about list of loaded modules. In sampling view module name will be empty if samples end up in such a librarly.

Reply:
From the code, I'd assume that they were intended to be filtered out. However, I am also fine with keeping them.

Reply:
Okay, so I'll move the `insert_or_assign` above the `if (elf_file->GetBuildId().empty()) continue;`. So the handling of cases where the elf_file has no build_id would be handled as before.